File: BOJ_1357/BOJ_1357.java
import java.io.*;
import java.util.*;
class Main {
	public static void main(String[] args) 
	throws IOException{
		BufferedReader BR = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter BW = new BufferedWriter(new OutputStreamWriter(System.out));
		StringBuilder SB = new StringBuilder();
		String[] XY = BR.readLine().split(" ");
		int X = FLIP(Integer.toString(FLIP(XY[0])+FLIP(XY[1])));
		BW.write(X+"\n");
		BW.flush();
	}
	public static int FLIP(String S){
		StringBuffer TEMP = new StringBuffer(S);
		String ANS = TEMP.reverse().toString();
		int REINT = Integer.parseInt(ANS);
		return REINT;
	}
}

File: BOJ_1357/BOJ_1357.cs
using System;
using System.Linq;
using System.Text;
namespace asharien{
    static class PROGRAM{
        static void Main(){
            int[] XY = Console.ReadLine().Split().Select(int.Parse).ToArray();
            String X = XY[0].ToString();
            String Y = XY[1].ToString();
            Console.WriteLine(FLIP((FLIP(X)+FLIP(Y)).ToString()));

        }
        public static int FLIP(this string S){
            char[] X = S.ToCharArray();
            Array.Reverse(X);
            int ANS = Convert.ToInt32(new string(X)); 
            return ANS;
        }
    }
}
